packages/react/src/progress-indicator/ProgressIndicator.test.tsx

```diff
@@ -1,18 +1,68 @@
 import '@testing-library/jest-dom';
 import 'html-validate/jest';
 import { axe, toHaveNoViolations } from 'jest-axe';
-import { render, cleanup } from '../../../../test-utils';
+import { render, screen } from '../../../../test-utils';
 import { ProgressIndicator, ProgressIndicatorProps } from './ProgressIndicator';
 import {
-	ProgressIndicatorItem,
-	ProgressIndicatorItemStatus,
 	activeProgressStatusAllowedList,
 	allProgressStatuses,
-} from './ProgressIndicatorItem';
+	ProgressIndicatorItemStatus,
+	ProgressIndicatorItem,
+} from './utils';
 
 expect.extend(toHaveNoViolations);
 
-afterEach(cleanup);
+function createList(
+	activeStatus: ProgressIndicatorItemStatus,
+	extraProperties: Partial<ProgressIndicatorItem>
+): ProgressIndicatorProps {
+	return {
+		items: [
+			{
+				...extraProperties,
+				label: 'Done',
+				status: 'done',
+				isActive: activeStatus === 'done',
```

Comment:
This feels like `isActive` should only be set in the legacy tests

Reply:
It is. Are you saying we should move the function back into the test?

Reply:
Oh, I missed the nesting. Can we call it `createLegacyList` then? Or move it into the legacy section?

Reply:
- [x] Rename `createList` to `createLeagacyList`

Reply:
>Oh, I missed the nesting.

This is why I love `Given-when-then`. A consistent structure is easier to reason about.